src/questionnaireitem_media_video.js

```diff
@@ -20,6 +20,8 @@ function QuestionnaireItemMediaVideo(className, question, required, url, readyOn
 
     TheFragebogen.logger.debug(this.constructor.name + "()", "Set: className as " + this.className + ", urls as " + this.height + ", width as " + this.width);
 
+    this.videoSource = null;
+    this.pTag = null;
```

Comment:
Not requiried as class attribute.
It is only used in createUI().

Reply:
Removed it.